### PR TITLE
Refactoring mutual.py using versus_test

### PR DESCRIPTION
**Pull request by MariosZoulias**
#884

### CODE REVIEW

File: axelrod/tests/strategies/test_mutual.py [hunk 1]
```diff
@@ -22,10 +22,37 @@ class TestDesperate(TestPlayer):
     def test_strategy(self):
         self.first_play_test(C, seed=1)
         self.first_play_test(D, seed=2)
-        self.responses_test([D], [C] * 4, [D] * 4)
-        self.responses_test([D], [D, D, C], [C, C, D])
-        self.responses_test([C], [D, D, D], [C, C, D])
 
+        #Our Player (Desperate) vs Cooperator SEED -->1
```
Comment (Nikoleta-v3) [inline axelrod/tests/strategies/test_mutual.py]: Hello @MariosZoulias. I just have a few comments on PEP8. PEP8 is a style guide for python code, so it's just tips on making the code look nice and readable :smile: . There should a space after `#` and before the comment and I would add a space after `-->` as well. Thus, `# Our Player (Desperate) vs Cooperator SEED --> 1`. This applies for all the comments below.

File: axelrod/tests/strategies/test_mutual.py [hunk 2]
```diff
@@ -22,10 +22,37 @@ class TestDesperate(TestPlayer):
...
+        opponent = axelrod.Cooperator()
+        opponent_actions = [C] * 5
+        actions = [(C, C), (D, C), (D, C), (D, C), (D,C)]
```
Comment (Nikoleta-v3) [inline axelrod/tests/strategies/test_mutual.py]: Also in tuples a space after the comma is the stylish way to go  `(D, C)`.

File: axelrod/tests/strategies/test_mutual.py [hunk 3]
```diff
@@ -22,10 +22,37 @@ class TestDesperate(TestPlayer):
...
+        self.versus_test(opponent, expected_actions=actions, seed=1)
+
+        #Our Player (Desperate) vs Cooperator SEED -->2
+        opponent = axelrod.Cooperator()
+        actions = [(D, C), (D, C), (D, C), (D, C), (D,C)]
```
Comment (Nikoleta-v3) [inline axelrod/tests/strategies/test_mutual.py]: space after comma

File: axelrod/tests/strategies/test_mutual.py [hunk 4]
```diff
@@ -22,10 +22,37 @@ class TestDesperate(TestPlayer):
...
+        self.versus_test(opponent, expected_actions=actions, seed=2)
+
+        #Our Player (Desperate) vs Defector SEED -->1
+        opponent = axelrod.Defector()
+        actions = [(C, D), (D, D), (C, D), (D, D), (C,D)]
```
Comment (Nikoleta-v3) [inline axelrod/tests/strategies/test_mutual.py]: same here and so on ...

File: axelrod/tests/strategies/test_mutual.py (hunk 1 above)
Comment (MariosZoulias) [inline axelrod/tests/strategies/test_mutual.py]: Thank you Nikoleta ;)

**Comment by MariosZoulias** [comment]
Of course , am going to do that in a few hours .
Thank you ;)

File: axelrod/tests/strategies/test_mutual.py [hunk 5]
```diff
@@ -20,12 +20,39 @@ class TestDesperate(TestPlayer):
     }
 
     def test_strategy(self):
-        self.first_play_test(C, seed=1)
-        self.first_play_test(D, seed=2)
-        self.responses_test([D], [C] * 4, [D] * 4)
-        self.responses_test([D], [D, D, C], [C, C, D])
-        self.responses_test([C], [D, D, D], [C, C, D])
+        self.first_play_test(C, seed = 1)
```
Comment (marcharper) [inline axelrod/tests/strategies/test_mutual.py]: For function arguments we don't include spaces around equal signs:
`self.first_play_test(C, seed=1)`

That's the guideline in PEP8. There are several more instances below.

Comment (MariosZoulias) [inline axelrod/tests/strategies/test_mutual.py]: Ok thank you . Let me re-fix it ;)

**Pull request merged**